### PR TITLE
fixed infowindow for many maps with a single marker in a single page

### PR DESCRIPTION
**Pull request by GermanMtzmx**
fix to set correct location of infowindow when there are many maps with infowindow enabled in the same page

### CODE REVIEW

File: GoogleMapHelper.php [hunk 1]
```diff
@@ -255,11 +255,22 @@ function setMarker(map, id, position, title, icon, shadow, content, showInfoWind
             });
            if(content != '' && showInfoWindow){
              var infowindow = new google.maps.InfoWindow({
-                  content: content
+                  content: content,
```
Comment (marcferna) [inline GoogleMapHelper.php]: Remove extra comma and extra lines

File: GoogleMapHelper.php [hunk 2]
```diff
@@ -255,11 +255,22 @@ function setMarker(map, id, position, title, icon, shadow, content, showInfoWind
...
+                  
+
               });
-             google.maps.event.addListener(markers[index], 'click', function() {
-            infowindow.open(map,markers[index]);
+            google.maps.event.addListener(markers[index], 'click', function() {
+              // fix to show infowindow in the correct location for many maps in a single place 
+              //old code => infowindow.open(map,markers[index]);
```
Comment (marcferna) [inline GoogleMapHelper.php]: No need of this

File: GoogleMapHelper.php [hunk 3]
```diff
@@ -255,11 +255,22 @@ function setMarker(map, id, position, title, icon, shadow, content, showInfoWind
...
+
+              for(w=0;w<markers.length;w++){
+               
+                infowindow.setContent(content);
+                infowindow.open(map,this)
+              }
+
               });
+
```
Comment (marcferna) [inline GoogleMapHelper.php]: Remove extra lines

**Comment by marcferna** [comment]
Thanks for your work @GermanMtzmx ! Please, see comments, update it and i'll merge it.

**Comment by GermanMtzmx** [comment]
@marcferna i already update my fork, should i open a new pull request im new on github :V sorry for my bad english

**Comment by marcferna** [comment]
No worries! Let's reopen this one so we have all the information here. I'll close the other one.

Thanks again!

**Comment by GermanMtzmx** [comment]
@marcferna  i closed the first pull request and open a new one with the changes, greets from mexico and its a nice plugin

**Comment by marcferna** [comment]
I merged it

**Pull request closed without merging**